lingcorpora/corpora/rus_poetic_corpus.py

```diff
@@ -0,0 +1,727 @@
+        Затянутая(любая 5- или 6-строчная строфа на две рифмы): затянутая
+        Цепная(терцины или подобная): цепная
+        Сложная(иные типы): сложная
+        ааб ввб
+        абааб
+        абаб вв
+        абаб ввгг
+        абаб вггв
+        абабаб
+        абабаб вв
+    Дополнительные параметры (doc_extra):
+        Полиметрия:
+            полиметрия
+            полиметрический фрагмент
+            с полиметрическими фрагментами
+            скользящий переход
+            упрощенная разметка
+        Преобразования метра:
+            перебои
+            цезурные наращения
+            цезурные усечения
+            переменная анакруса
+            нарушения анакрусы
+            нарушения внутренней анакрусы
+            урегулированная анакруса
+            сверхдлинная анакруса
+            нарушение ударной константы
+        Особенности строфики:
+            нарушения строфики
+            усеченная строка
+            холостая строка
+            белая строка
+            кода
+            рекурсивная строфика
+            рефрен
+            редупликация
+            строчный логаэд
+            прозаические фрагменты
+            магистрал
+        Особенности рифм:
+            тавторифма
+            омонимическая рифма
+            монотонная рифма
+            внутренняя рифма
+            разноударная рифма
+            неравносложная рифма
+            составная рифма
+            ассонанс
+            диссонанс
+            корневая рифма
+            начальная рифма
+        Особенности метров:
+            пеон I
+            пеон II
+            пеон III
+            пеон IV
+            пентон
+            гиперпентон
+            пеонический дольник
+            логаэдический дольник
+        Прочее:
+            внутрисловный перенос
+    Общая формула (doc_formula)
+    Формула для поиска по строке (s_sp_frm_sch):
+        Формула строки совпадает со стандартной стиховедческой записью: метр + количество стоп/иктов/слогов + клаузула, например, Я6д.
+        Также тонические и силлабо-тонические строки можно искать по схеме расположения иктов, например, 0*2*1*1*0
+    Имя автора (doc_author_id)
+    Пол автора(doc_sex):
+        муж
+        жен
+"""
+
+TEST_DATA = {'test_single_query': {'query': 'мост'},
+             'test_multi_query': {'query': ['мост', 'скала']}
+             }
+
+
+class PageParser(Container):
+    def __init__(self, *args, stress=False, exact=True, markup=False, get_lines=False, get_text=True, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.__page = 0
+        self.__stress = stress
+        self.__exact = exact
+        self.__markup = markup
+        self.__get_lines = get_lines
+        self.__get_text = get_text
+
+        self.__pattern = re.compile('(\s*[,?!\.:;―\-«»\"\'\'\[\]\(\)…]+\s*)')
+
+        self.__subcorp_url = 'https://processing.ruscorpora.ru/search.xml?env=alpha&mode=poetic&sort=i_grtagging&lang=ru&spd=1&text=meta&is_subcorpus=1&dpp=50&'
```

Comment:
Could you please move the common part of these URLs (`https://processing.ruscorpora.ru`) to a separate variable? If the domain changes, we will have to change it in one place.